app/services/arcgis_api/mock/responses/request_suggestions_response.json

```diff
@@ -0,0 +1,29 @@
+{
+  "suggestions": [
+      {
+          "text": "210 Humboldt Avenue, Ellinwood, MA, 02121, USA",
+          "magicKey": "dHA9MCNsb2M9MTQ3NTc5MjQjbG5nPTU0I2ZhPTMwODAxOTIjaG49MjEwI2xicz0xMDk6NjAyMjI1Njc=",
+          "isCollection": false
```

Comment:
Did you check cases where the API returns a value with `isCollection` set as `true`? We may need to treat the collection values differently if the current parameters are still returning them.

Reply:
> Since we scope the category to only address, it will only suggest non-collection unique addresses.

The documentation and behavior suggests that this is the case, but it'd be nice if it was formally spelled out.